jest.preset.js

```diff
@@ -18,6 +18,7 @@ module.exports = {
     '!**/*.config.*',
     '!**/infra/*',
     '!**/seeders/*',
-    '!**/migration/*',
+    '!**/migrations/*',
+    '!**/{buildOpenApi,openApi,main,index}.ts',
```

Comment:
Some files include the actual logic in `index.ts` and those should be tested + collected for coverage.
Most of them are re-exports to define the API of the domain. So we can't safely ignore all of them.

Reply:
```suggestion
    '!**/{buildOpenApi,openApi,main}.ts',
```
What about `main.ts`?

Reply:
That's fine, since it's not targeting `main.tsx` files. The other ones are just bootstraps, which is OK to ignore, but not mandatory :D.

Reply:
Nah, thinking again, we should skip it since it could leave us in a bad debug trying to figure out why a coverage isn't taking affect on valid files.

Reply:
Can you explain further? Can bootstrapping even be unit/integration tested? Should bootstrapping be tested in e2e (system testing)?

Reply:
We can't be 100% certain that logic in all `main.ts` files will all be the island.is nestjs bootstrap.

Reply:
What about `buildOpenApi.ts` and `openApi.ts`? Are there more files which don't contain logic or are only for bootstrapping/setup?